### DISCUSSION TITLE
Objective statement

### DISCUSSION THREAD

**Discussion by ottomorac** [veramolabs/did-eth#6] [General]
Our objective is to collaboratively design and come up with a new did method proposal that is generally compatible with ethereum compatible blockchains. It is critical that the did:method be collaboratively designed and governance be done by a neutral entity in order to maximize the possibility of wide adoption and potential for standardization across EVM compatible chains.

**Comment by Hmac512** [comment]
I’d like to add that we should aim for the same contract to deployed across all EVM chains, so we don’t get into a situation like we’re in now.

Also would make bridging easier if it’s decided to add support for it.

it’ll keep the ecosystem clean, and that will pay dividends for the developer experience.

I’m imagining the governance will be responsible for making sure everything is consistent across all chains.

**Reply by bumblefudge** [reply]
"all chains" or "all L2s" is a deceptively self-evident concept.  I'd drill down a bit on which chains are "all chains" (other L1s in EVM mode? StarkNet or other L1-anchored L2s with their own smart contract language? L2s that fully support a finite list of core EVM methods without caveats?).

**Reply by bumblefudge** [reply]
also, deployÈD or deployABLE?

**Reply by mirceanis** [reply]
Good points.

I think that as long as the contracts are upgradable it would not be too hard to avoid the problems we've seen with did:ethr:
* frozen contracts, causing difficulty migrating to a new variant
* contracts diverging between deployments on different chains

I agree that there is no "all chains". Would a list of deployments be enough?
